doc/bips.md

```diff
@@ -1,4 +1,4 @@
-BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.10.0**):
+BIPs that are implemented by Bitcoin Classic (up-to-date up to **v0.10.0**):
```

Comment:
This is fine until we have a definitive version number for first version i think